### PR TITLE
AT_01.17.01 | Verify `X` button is visible, clickable and closing Popup

### PR DESCRIPTION
**Pull request by Sviatlana29**

### CODE REVIEW

**Comment by Sviatlana29** [comment]
[AT_01.17.01 | Restore password Popup(from Login popup) > Verify `X` button is visible, clickable and closing Popup](https://trello.com/c/INpbRGng/240-at011701-restore-password-popupfrom-login-popup-verify-x-button-is-visible-clickable-and-closing-popup)

File: cypress/e2e/testUiAndFunction/startPage/restorePasswordPopupSection/us_01.17_HeaderElements.cy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    const AGENT = Cypress.env('agent');
+
+    beforeEach(function () {
+        cy.fixture('startPage/alert').then(alert => {
+            this.alert = alert;
+        });
+        cy.visit('/');
+        startPage.clickLoginButton();
+        loginPopup.clickForgotYourPasswordLink();
+        restorePopup.getRestorePopup().should('be.visible')
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/startPage/restorePasswordPopupSection/us_01.17_HeaderElements.cy.js:22]: All assertions must be in the test(it)

Comment (Sviatlana29) [reply cypress/e2e/testUiAndFunction/startPage/restorePasswordPopupSection/us_01.17_HeaderElements.cy.js:22]: Done

**Review by Rom-T — APPROVED** [review]
LGTM

**Pull request merged**